task11/6.sql

```diff
@@ -0,0 +1,4 @@
+SELECT USER.USER_ID, NAME, COUNT(USER.USER_ID) from USER
+JOIN POST on POST.USER_ID = USER.USER_ID
+WHERE CREATED_AT like '%03-01%'
```

Comment:
Well.. okay) Most people fall back to likes of DATE(2020-03-01), but you've taken this most literally and are somewhat right)